genesyscloud/architect_datatable/genesyscloud_architect_datatable_init_test.go

```diff
@@ -20,11 +20,15 @@ type registerTestInstance struct {
 
 // registerTestResources registers all resources used in the tests
 func (r *registerTestInstance) registerTestResources() {
+	r.resourceMapMutex.Lock()
```

Comment:
These were missed during the code review.  We should be locking the map so we avoid any potential concurrent map issues,

Reply:
Was it the linter that caught this?

Reply:
It caught that the variables were unused.   Once I knew they were unused, I realized that we need to include the locking in there.